aptos-move/framework/aptos-framework/sources/native_bridge.move

```diff
@@ -585,6 +719,44 @@ module aptos_framework::native_bridge {
         assert!(bridge_relayer() == new_relayer, 0);
     }
 
+    #[test(aptos_framework = @aptos_framework)]
```

Comment:
shouldn't testing be in a separate file?

Reply:
Not necessary

Reply:
not a requirement